playwright.config.js

```diff
@@ -0,0 +1,76 @@
+const { defineConfig } = require('@playwright/test');
+const path = require('path');
+
+module.exports = defineConfig({
+  testDir: path.join(__dirname, 'src', 'test', 'e2e', 'tests'),
+  timeout: 60000,
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  globalTeardown: require.resolve('./src/test/e2e/global-teardown.js'),
+  // Increase workers in CI - adjust based on your Jenkins node capacity
+  workers: process.env.CI ? 4 : 4,
+
+  reporter: [
+    ['html'],
+    ['list'], // Added for better CI console output
+  ],
+
+  use: {
+    baseURL: process.env.CI ? process.env.TEST_URL : 'http://localhost:3300',
+    trace: 'on-first-retry',
```

Comment:
Would retain-on-failure be better? 
Might be the case we want logs for each retry...although It could be the case that it'll literally be the same error

Will take your view on this. If you feel it could be different perhaps change. I would imagine another option would be last retry? (if it exists).

Reply:
I have a feeling retaining all the retry logs would probably be actively detrimental to speed of fixes as you'd get a huge wave of logs with a lot of the same problem over and over while trying to write. However, not got particularly strong views either way.

In fact, may be best to link it to process.env.CI to see fewer logs locally?